utils/rjson.hh

```diff
@@ -55,7 +55,7 @@ public:
 // 2. assert() crashes a program
 // Fortunately, the default policy can be overridden, and so rapidjson errors will
 // throw an rjson::error exception instead.
-#define RAPIDJSON_ASSERT(x) do { if (!(x)) throw rjson::error(std::string("JSON error: condition not met: ") + #x); } while (0)
+#define RAPIDJSON_ASSERT(x) do { if (!(x)) throw rjson::error(fmt::format("JSON assertion failed at: {}", current_backtrace_tasklocal())); } while (0)
```

Comment:
The condition itself may also be useful, although after grepping, I see the only uses of this macro are `RAPIDJSON_ASSERT(false)`, so indeed printing the condition doesn't give much in these cases

Reply:
Yes, I only ever saw `false` printed as the condition.